src/main/webapp/comments.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      type="text/javascript"
+      src="https://www.gstatic.com/charts/loader.js"
+    ></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <p>Hello</p>
+    <div id="hours-chart" style="width: 900px; height: 500px"></div>
+    <div id="difficulty-chart" style="width: 900px; height: 500px"></div>
+
+    <p>Rate your term and professors!</p>
+    <form action="/data" method="POST">
+      <p>Rate your term:</p>
+
+      <textarea name="term-input" maxlength="1500">
+Write opinion on term here.</textarea
+      >
+
+      <p>How would you rate this term?</p>
+      <input
+        type="number"
+        id="rating-term"
+        name="rating-term"
+        min="0"
+        max="5"
+      />
+      <label for="rating-term"
+        >Term Rating (From 0 = Terrible to 5 = Excellent)</label
```

Comment:
For longer content like this we often put the html tags on their own lines, which would probably also fix the weird prettier formatting.